packages/hash/shared/src/blockMeta.ts

```diff
@@ -47,52 +47,95 @@ export const componentIdToUrl = (componentId: string) =>
   componentId.replace(/\/$/, "");
 
 /**
- * transform mere options into a useable block configuration
+ * Get an absolute url if the path is not already one.
  */
-const toBlockConfig = (
-  options: BlockMetadata,
-  componentId: string,
-): BlockConfig => {
+function getAbsoluteUrl(args: { baseUrl: string; path: string }): string;
+function getAbsoluteUrl(args: {
+  baseUrl: string;
+  path?: string | null | undefined;
+}): string | null | undefined;
+function getAbsoluteUrl({
```

Comment:
Strictly speaking, we’re not getting a URL but constructing it. There’s now absolute URL anywhere in the memory till we’ve called this function. Not suggesting to make changes just to rename a local function, just paying attention to help future PRs. I’d use `generateAbsoluteUrl` / `deriveAbsoluteUrl` or something like that.

Also `args` could be renamed into `payload`. Args is plural and is often used in `(a, b, ...args)`.

Reply:
https://github.com/hashintel/hash/pull/481/commits/e63d694f6833fa786820bf39edb995af90a603bd